### DISCUSSION TITLE
Support node cluster for global page files render.

### DISCUSSION THREAD

**Discussion by Buminta** [vikejs/vike#1197] [Discussion]
### Description

When I'm used node cluster for run node in multi cores.
Howerver application slow load pages files before render page.

```const pageContext = await renderPage(pageContextInit)```
In here usually process in 2-3s. And when debug i see slow from load PageFiles!

**Comment by brillout** [comment]
Moving to GitHub Discussions as it isn't clear that it's related to Vike.

What makes you think it's related to Vike?
